schemas/request-metadata-property.json

```diff
@@ -0,0 +1,29 @@
+{
+    "type": "object",
+    "additionalProperties": false,
```

Comment:
3c480518bb99b111cb8a477e160f25e4cdb108d1 and the docs 5fdb0172aa9c9e34b69e97e87c2c94cc07771fa1